frontend/src/components/TaskCalendar.jsx

```
@@ -130,7 +687,12 @@ const TaskCalendar = () => {
     setNewTask(newTask);
   };
 
-  const handleSelectEvent = (event) => {
+  const handleSelectEvent = (event, e) => {
+    // Use the click event coordinates
+    if (e && e.pageX && e.pageY) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)